api/lecture/controllers/lecture.js

```diff
@@ -1,23 +1,27 @@
 'use strict';
 const { sanitizeEntity } = require('strapi-utils');
 
-const {
+/* const {
   isValidYoutubeUrl,
   getVideoId,
   getVideoDuration,
   getPlaylistContents,
   getPlayListId,
-} = require('../../helpers/youtube');
+} = require('../../helpers/youtube'); */
+
+const { getId, getSingleVideo, getPlaylistContents } = require('../../helpers/vimeo');
```

Comment:
we don't need to get rid of youtube stuff, we can leave both together, right? we can just decide which to use based on URl